recipes/ibm_db_django/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "ibm_db_django" %}
+{% set version = "1.4.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b7e504d84a51760729e16a7afc4942c535a05e1f775f96cbd33fb217e677a4e
+
+build:
+  number: 0
```

This looks like it could be noarch?
```suggestion
  noarch: python
  number: 0
```

---

Added the noarch to check the impact